app/main/system_message.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from app.libs.utils import convert_tx_id
+from app.metadata.metadata_store import MetaDataStore
+
+from flask import request
+from flask.ext.themes2 import render_theme_template
+
+from flask_login import current_user
+
+from ua_parser import user_agent_parser
+
+logger = logging.getLogger(__name__)
+
+
+def system_message(message):
```

Comment:
Codecov is reporting no test coverage of this function, this implies we have no component test to verify this is working, see https://codecov.io/gh/ONSdigital/eq-survey-runner/src/0539aa4bce732111df8de49221be6a741b33c909/app/main/system_message.py